Arcemi.Pathfinder.Kingmaker/DictionaryAccessor.cs

```diff
@@ -46,9 +46,7 @@ void IModelContainer.Refresh()
         public T this[string key]
         {
             get => _dict[key];
-            set {
-                throw new NotImplementedException();
-            }
+            set => _dict[key] = value;
```

Comment:
Need to set data on the underlying object, though from what I can see you're not using the setter anywhere, so it's fine, I would still leave it as before with a thrown exception instead of causing a misunderstanding that the value would be set.

Reply:
Good call. In my experimental branch I took out the setter entirely.